.github/workflows/publish.yml

```diff
@@ -95,13 +95,34 @@ jobs:
       - name: Verify demo browser npm version and run deployment script
         run: |
           demo_current_version=$(.github/script/get-demo-browser-current-version)
+          demo_current_version=$(echo $demo_current_version | tr -d '^~*')
           echo "Current demo version:" $demo_current_version
+
+          echo "Current released npm version:" ${{ steps.npm_version.outputs.npm_version }}
           if [[ *$demo_current_version* != "^#${{ steps.npm_version.outputs.npm_version }}" ]];
           then
             echo "Demo npm version is not the same as the current released version"
             exit 1
           else
             npm run build --app=meetingReadinessChecker
             script/github-action-awscli-installation
-            script/deploy-canary-demo
+            
+            npm_latest_version=$(npm view amazon-chime-sdk-js@latest version)
+            echo "npm_latest_version:" $npm_latest_version
+            npm_beta_version=$(npm view amazon-chime-sdk-js@beta version)
+            echo "npm_beta_version:" $npm_beta_version
+
+            if [[ $npm_beta_version == $npm_latest_version || -z $npm_beta_version ]] ; then
+              echo "Currently, there is no beta version or the beta and latest version are same. So, the demo will be deployed on the latest CloudFormation stack."
+              script/deploy-canary-demo
```

Comment:
Don't think this is correct. I assume when we release 3.0.0 we still want to run canary for 2.0.0 up to a certain time.

Reply:
Imo, logic should be:
```
latestVersion = max(npm_beta, npm_latest).
if (getMajorVersion(latestVersion) === getMajorVersion(deployedVersion)) {
   script/deploy-canary-demo
} else if (... >= )) {
   script/deploy-canary-demo prev
} else  {
   error ...
}
```

Reply:
In the case, where there is no beta version on `npm` and we still want to run PMV canary this logic will run into an issue. Let me think about this scenario.